x-pack/plugins/cases/public/components/all_cases/table_filters.tsx

```diff
@@ -59,47 +68,42 @@ const CasesTableFiltersComponent = ({
   availableSolutions,
   displayCreateCaseButton,
   onCreateCasePressed,
+  isLoading,
+  currentUserProfile,
 }: CasesTableFiltersProps) => {
-  const [selectedReporters, setSelectedReporters] = useState(
-    initial.reporters.map((r) => r.full_name ?? r.username ?? '')
-  );
   const [search, setSearch] = useState(initial.search);
   const [selectedTags, setSelectedTags] = useState(initial.tags);
   const [selectedOwner, setSelectedOwner] = useState(initial.owner);
+  const [selectedAssignees, setSelectedAssignees] = useState<UserProfileWithAvatar[]>([]);
+  const fetchAssignees = useRef<() => void>();
   const { data: tags = [], refetch: fetchTags } = useGetTags(CASE_LIST_CACHE_KEY);
-  const { reporters, respReporters, fetchReporters } = useGetReporters();
+
+  const queryClient = useQueryClient();
 
   const refetch = useCallback(() => {
     fetchTags();
-    fetchReporters();
-  }, [fetchReporters, fetchTags]);
+    queryClient.refetchQueries([USER_PROFILES_CACHE_KEY, USER_PROFILES_SUGGEST_CACHE_KEY]);
```

Comment:
I think the cache key should be `[USER_PROFILES_CACHE_KEY, USER_PROFILES_BULK_GET_CACHE_KEY]`